package.json

```diff
@@ -48,6 +48,7 @@
     "ember-cli-test-loader": "^3.0.0",
     "ember-cli-typescript-blueprints": "^3.0.0",
     "ember-cli-uglify": "^3.0.0",
+    "ember-cli-version-checker": "^5.1.0",
```

Comment:
This should be in `dependencies` not `devDependencies` (this is likely also why the eslint rule is failing)

Reply:
Fixed.